src/radical/pilot/configs/resource_local.json

```diff
@@ -139,6 +138,61 @@
         "virtenv"                     : "%(global_sandbox)s/ve_localhost",
         "virtenv_mode"                : "create",
         "python_dist"                 : "anaconda"
-    }
-}
+    },
 
+    "localhost_orte": {
+        "description"                 : "Your local machine.",
+        "notes"                       : "To use the ssh schema, make sure that ssh access to localhost is enabled.",
+        "schemas"                     : ["local", "ssh"],
+        "ssh"                         :
+        {
+            "job_manager_endpoint"    : "ssh://localhost/",
+            "filesystem_endpoint"     : "sftp://localhost/"
+        },
+        "local"                       :
+        {
+            "job_manager_endpoint"    : "fork://localhost/",
+            "filesystem_endpoint"     : "file://localhost/"
+        },
+        "default_remote_workdir"      : "$HOME",
+        "lrms"                        : "FORK",
+        "agent_type"                  : "multicore",
+        "agent_scheduler"             : "CONTINUOUS",
+        "agent_spawner"               : "POPEN",
+        "agent_launch_method"         : "ORTE",
+        "task_launch_method"          : "ORTE",
+        "mpi_launch_method"           : "ORTE",
+        "rp_version"                  : "debug",
+        "virtenv"                     : "%(global_sandbox)s/ve_localhost",
+        "virtenv_mode"                : "create",
+        "python_dist"                 : "default"
+    },
+
+    "localhost_ortelib": {
+        "description"                 : "Your local machine.",
+        "notes"                       : "To use the ssh schema, make sure that ssh access to localhost is enabled.",
+        "schemas"                     : ["local", "ssh"],
+        "ssh"                         :
+        {
+           "job_manager_endpoint"    : "ssh://localhost/",
+           "filesystem_endpoint"     : "sftp://localhost/"
+        },
+        "local"                       :
+        {
+           "job_manager_endpoint"    : "fork://localhost/",
+           "filesystem_endpoint"     : "file://localhost/"
+        },
+        "default_remote_workdir"      : "$HOME",
+        "lrms"                        : "FORK",
+        "agent_type"                  : "multicore",
+        "agent_scheduler"             : "CONTINUOUS",
+        "agent_spawner"               : "ORTE",
+        "agent_launch_method"         : "ORTE_LIB",
+        "task_launch_method"          : "ORTE_LIB",
```

Comment:
after what we have seen on titan, should that also be `orte/orte/orte_lib/orte_lib`, so that it doesn't break under certain agent configs?